source/server_write_commands.rst

```diff
@@ -139,15 +139,10 @@ ensure a batch can be correctly processed, two limits must be respected.
 Both of these limits can be found using isMaster():
 
 * ``maxBsonObjectSize`` : currently 16MB, this is the maximum size of writes (excepting command overhead)
-  that should be sent to the server.  Documents to be inserted, query documents for updates and 
+  that should be sent to the server.  Documents to be inserted, query documents for updates and
```

Comment:
How about, "The sum of the sizes of documents in the batch must be <= this size. That is, the sum of the sizes of all documents to be inserted, query documents for updates and deletes, or the update expression documents within a write command must not exceed maxBsonObjectSize. Once these documents have been assembled into a write command, the write command's total size may exceed maxBsonObjectSize by a small amount."

It's tempting to delete the extra info, as you did, but I don't want the next generation to read this, ask, "But how can I insert a 16MB document?" and begin the cycle anew.